### ISSUE TITLE
Want to know how to run this project

### ISSUE THREAD

**Issue by affanmujawar25** [codingelle/django-whatsapp-web-clone#29]
I followed the steps which you have mentioned in README.md file. then also i am not able to run this project. Please Help me to run this project. i have attached the error screenshots
![Screenshot 2024-01-23 215437](https://github.com/codingelle/django-whatsapp-web-clone/assets/103987133/da989e24-c9f5-4b02-a804-c56da61e5947)
![Screenshot 2024-01-23 215450](https://github.com/codingelle/django-whatsapp-web-clone/assets/103987133/d1608179-7328-4ff7-9ff4-dc89866bc083)

**Comment by josnin** [comment]
@affanmujawar25  make sure to install all the required python package successfully

**Comment by affanmujawar25** [comment]
I tried it. But some packages are not getting installed.

**Comment by josnin** [comment]
> I tried it. But some packages are not getting installed.

Can you try using and installing the required package with Python 3.7? In your screenshot, it shows Python 3.12, and I haven't tested it with Python 3.12 yet.

**Comment by affanmujawar25** [comment]
Okay I will Try it with Python 3.7

Actually I'm pushing Master's in Computer Applications. So we are having Django as a Elective Subject it's a "project based learning " so I took The real time chat app as a final course project. Then I found your this project in YouTube.

**Comment by affanmujawar25** [comment]
> I tried it. But some packages are not getting installed.



> > I tried it. But some packages are not getting installed.
> 
> Can you try using and installing the required package with Python 3.7? In your screenshot, it shows Python 3.12, and I haven't tested it with Python 3.12 yet.

**Comment by affanmujawar25** [comment]
> > I tried it. But some packages are not getting installed.
> 
> Can you try using and installing the required package with Python 3.7? In your screenshot, it shows Python 3.12, and I haven't tested it with Python 3.12 yet.
> 

Okay I will Try it with Python 3.7

Actually I'm pushing Master's in Computer Applications. So we are having Django as a Elective Subject it's a "project based learning " so I took The real time chat app as a final course project. Then I found your this project in YouTube and I much need it because the first phase of project demonstration has been Scheduled next week.

I tried it with Python 3.7, in that "uvloop>=0.18.0" is not being installed
